corehq/apps/app_manager/views/download.py

```diff
@@ -408,21 +407,24 @@ def validate_form_for_build(request, domain, app_id, form_unique_id, ajax=True):
 
 
 def download_index_files(app, build_profile_id=None):
+    def needed_for_ccz(path):
+        if profiles:
```

Comment:
Looks like `profiles` is undefined if `build_profile_id is not None`, which seems like it would result in a name error. Does that sound right?

Reply:
It sure does! I should have caught this. I suppose I could initialize it to None where `prefix` is set to a default value.

Reply:
48a0b4a95e4bf1a3764ccb48678874f2bbece0d9